README.md

```diff
@@ -9,24 +9,24 @@
 
 Ping provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+Our samples showcase functionality of our SDKs to get you up and running,
+whether you are using PingOne Advanced Identity Cloud,
+PingAM, OIDC Login, or PingOne DaVinci (coming soon), we've got you covered.
+
+Explore the many use cases
+the Ping SDKs have to offer by referring to each respective SDK:
 
-We provide samples of integrating with PingOne Advanced Identity Cloud and PingAM for the following platforms:
 - [JavaScript](./javascript/)
 
 - [iOS](./iOS/)
 
 - [Android](./android/)
 
-### Integrate with PingOne DaVinci:
-
-- Samples coming soon
-
 ## Documentation
 
 Detailed [documentation](https://docs.pingidentity.com/sdks/latest/sdks/index.html) is provided, and includes topics such as:
```

Comment:
Just `https://docs.pingidentity.com/sdks` is probably safer/more robust to changes.